tests/e2e/support/utils/expirationDatePicker.ts

```diff
@@ -0,0 +1,24 @@
+export const addDays = (days: number): Date => {
```

Comment:
the file can be renamed to `datePicker.ts`. Is it specific to the expiration date picker?